docs/download_files.md

```diff
@@ -21,42 +22,109 @@ The files shown below will be included with each library (example shown for a li
 - A quality control report: `SCPCL000000_qc.html`,
 - A supplemental cell type report: `SCPCL000000_celltype-report.html`
 
+For more information on the contents of these files, see the sections on [gene expression data](#gene-expression-data), the [QC report](#qc-report), and the [cell type report](#cell-type-report).
+
 Every download also includes a single `single_cell_metadata.tsv` file containing metadata for all libraries included in the download.
+For a full description of the metadata file, refer to the [metadata section below](#metadata).
 
-Metadata-only downloads are also available, either by downloading the metadata for all samples in a single project or by downloading the metadata for all samples on the Portal.
-Please see the [section on metadata](#metadata) for a full description of the contents of the metadata files.
+Metadata-only downloads are also available, either by downloading the metadata for all samples in a single project using the `Download Sample Metadata` button or by downloading the [metadata for all samples on the Portal](#metadata-only-downloads).
 
-If downloading a project containing bulk RNA-seq data, two tab-separated value files, e.g., `SCPCP000000_bulk_quant.tsv` and `SCPCP000000_bulk_metadata.tsv`, will be included in the merged object download.
-The `SCPCP000000_bulk_quant.tsv` file contains a gene by sample matrix (each row a gene, each column a sample) containing raw gene expression counts quantified by Salmon.
-The `SCPCP000000_bulk_metadata.tsv` file contains associated metadata for all samples with bulk RNA-seq data.
+## Project downloads
 
-See also {ref}`processing bulk RNA samples <processing_information:Bulk RNA samples>`.
+To download the gene expression data for all samples in a single project, use the `Download Now` button next to the project title. 
+This will allow you to download the chosen project instantly, without having to wait for a download link to be available. 
```

Comment:
I'm not sure we need all this detail about the link, since the concept of having to wait for a download link hasn't introduced in the first place, so I'm suggesting to condense:

```suggestion
Use the `Download Now` button next to the project title to instantly download gene expression data for all samples in a single project.
```